config/runtime.exs

```diff
@@ -211,7 +219,8 @@ cond do
            :postgres_backend_adapter,
            filter_nil_kv_pairs.(
              url: System.get_env("POSTGRES_BACKEND_URL"),
-             schema: System.get_env("POSTGRES_BACKEND_SCHEMA")
+             schema: System.get_env("POSTGRES_BACKEND_SCHEMA"),
+             socket_options: [:inet6]
```

Comment:
should be a flag or from ip parsing

Reply:
can use :inet to check the type of address
https://www.erlang.org/doc/apps/kernel/inet.html#is_ipv4_address/1
maybe can have a helper function for this, would make the code clearer

Reply:
One problem is that `is_ipv4_address/1` only checks for the type of the IP passed in (whether the list of numbers has length 4, I think), so it must first be resolved and if there are both IPv4 and IPv6 available for a given host name, we must pick. It would actually be a lot better if the connection logic would determine the socket type and not the runtime configuration. I am convinced, you could implement that a lot quicker than me.

Reply:
Is now parsing the URL, resolving the host and determining IP type.